syntax = "proto3";

package catalog.category;

message CategoryMessage {
  uint32 feId = 1;
  string name = 2;
  string code = 3;
  uint32 parent = 4;
  uint32 sellerId = 5;
  bool isActive = 6;
  string taxInCode = 7;
  string taxOutCode = 8;
  string incomeAccount = 9;
  string internalIncomeAccount = 10;
  string tracking = 11;
  string stockValuationAccount = 12;
  string saleBranch = 13;
  string expenseAccount = 14;
  string returnAccount = 15;
  string path = 16;
}
